spec/support/inputs_outputs_pairs.rb

```diff
@@ -273,6 +273,17 @@ module JsonApi::Parameters::Testing
             genre_ids: [74]
           }
         }
+      ],
+      'long type name' => [
```

Comment:
Usually when the input-output test pairs are provided because of an issue I tried to leave a mark - such as just leaving a link within the input-output pair name. 
Could we follow this here as well? Something like: 

```
'long type name - type casing translation (https://github.com/visualitypl/jsonapi_parameters/pull/31)' => [...]
```